### ISSUE TITLE
I found some ocm secret commands which are not listed on ocm --help

### ISSUE THREAD

**Issue by mayankraipure12** [open-component-model/ocm#292]
**What would you like to be added**:
list all the possible commands in ocm cli help
eg: ocm oci  command is not visible in the help 
some more example ocm ocm
**Why is this needed**:
There are usecases which might be required to use it

**Comment by jensh007** [comment]
These seem to be duplicates somehow, but this is confusing. Is there any difference between 
`ocm oci artifacts download` and `ocm download artifact` for example? Why are there two commands?
Assignng this to @mandelsoft

**Comment by mandelsoft** [comment]
No, it just allows to use different order in the comannd spec. verb/object bs object/verb.

This is helpful for people used to a dedicated command scheme. Because these are no new command, they are omitted from the help.
It is just like autocorrection. for typos.

**Comment by Skarlso** [comment]
>No, it just allows to use different order in the comannd spec. verb/object bs object/verb.

The fact that different order of commands is possible is a bug in itself. This should _NOT_ be possible. It opens up so many possibilities of wrongfully calling something. And it's difficult to document proper availability and command structure.

It's also not intuitive at all. The fact the object/verb and verb/object do the same thing, is confusing rather than helpful. The structure of commands should be rigid. That would make it easy to construct trees or command chain and probably would simplify constructing commands and writing new ones significantly.
